src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -69,12 +69,12 @@ protected void doFilterInternal(final HttpServletRequest request,
                     timing,
                     contentEncoding,
                     acceptEncoding);
-
             nakadiKpiPublisher.publish(accessLogEventType, () -> new JSONObject()
                     .put("method", method)
                     .put("path", path)
                     .put("query", query)
-                    .put("app", Hex.encodeHexString(sha256MessageDigest.digest(user.getBytes(Charsets.UTF_8))))
+                    .put("app", user)
```

Comment:
I think that un-encrypted user data should have some prefix so that Data Lake automatically screens it.
@SergKam, should it be started with "user..." ?

Reply:
I don't know the rules of datalake detector, but user_id was automatically hidden as sensitive data.
Better ask guys from datalake, but they can mark it later manually anyway.